src/node_report_module.cc

```diff
@@ -67,8 +69,10 @@ void GetReport(const FunctionCallbackInfo<Value>& info) {
   GetNodeReport(env, "JavaScript API", __func__, error, out);
 
   // Return value is the contents of a report as a string.
-  info.GetReturnValue().Set(
-      String::NewFromUtf8(isolate, out.str().c_str()).ToLocalChecked());
+  Local<Value> ret;
+  if (ToV8Value(env->context(), out.str(), env->isolate()).ToLocal(&ret)) {
```

Comment:
Unrelated to this Pr, but we should definitely rewrite this function to avoid calling std::ostringstream